src/types/fetch.rs

```diff
@@ -24,16 +100,13 @@ pub struct Fetch {
     /// Only present if `RFC822.SIZE` was specified in the query argument to `FETCH`.
     pub size: Option<u32>,
 
-    // Note that none of these fields are *actually* 'static. Rather, they are tied to the lifetime
-    // of the `ZeroCopy` that contains this `Name`. That's also why they can't be public -- we can
-    // only return them with a lifetime tied to self.
-    pub(crate) fetch: Vec<AttributeValue<'static>>,
+    pub(crate) fetch: Vec<AttributeValue<'a>>,
     pub(crate) flags: Vec<Flag<'static>>,
 }
 
-impl Fetch {
+impl<'a> Fetch<'a> {
```

Comment:
Now that `Fetch` has a lifetime parameter, I'd like to have a `into_owned` method on it to get a `Fetch<'static>` from any `Fetch<'a>`.

Reply:
Done!